tools/assetAgent/src/main.cpp

```diff
@@ -558,7 +558,7 @@ static void processFileEntryDeltas(utArray<FileEntryDelta> *deltas)
         // Note we are using gActiveHandlers.size() outside of a lock, but this is ok as it's a word.
         if ((strstr(canonicalFile, ".loom") || strstr(canonicalFile, ".ls")) && (gActiveHandlers.size() > 0))
         {
-            lmLog(gAssetAgentLogGroup, "SENDING       -  %s", canonicalFile);
+            lmLog(gAssetAgentLogGroup, "Changed '%s'", canonicalFile);
```

Comment:
I think only displaying it as changed makes it more meaningful to the user. The fact that it's sending it over an asset agent connection seems to me more like an implementation detail than anything.

If you change a file, it makes more sense to see "Changed" pop up in the log rather than "Sending". Now sending does convey the additional meaning that the operation will take a bit to take hold, but that should be covered by the "Updating/Updated" messages that appear right after anyway.

After a couple more tweaks, the reload log looks like this now (first updating `credits.png` then `Thrusties.ls`):

```
     asset  Updated 'assets\credits.png', 78.3 KiB (80145 bytes)
     agent  Changed 'src\Thrusties.ls'
       cli  Recompiling 'src\Thrusties.ls'
     asset  Updated 'src\Thrusties.ls', 2.4 KiB (2445 bytes)
  compiler  LSC - Release JIT Compiler
  compiler  SDK: C:\Users\Miha\.loom\sdks\dev\
  compiler  Compiling: Main
  compiler  Linking: Feathers
  compiler  Linking: Loom
  compiler  Linking: System
  compiler  Compile successful: .\bin\Main.loom
     agent  Changed 'bin\Main.loom'
     asset  Updating 'bin\Main.loom', 0.8 MiB (858743 bytes)
     asset    12%
     asset    41%
     asset    71%
     asset  Updated 'bin\Main.loom', 0.8 MiB (858743 bytes)
       app  Reloading Main.loom
       app  ---------
       app  Thrusties
       app  ---------
     asset  Loaded 'assets/splash.png', 5.0 KiB (5141 bytes)
     asset  Loaded 'assets/bg.png', 2.7 KiB (2760 bytes)
```

What do you think?